frontend/styles/slider/gmd/index.js

```diff
@@ -3,8 +3,7 @@ import { themeConfig } from '@shopgate/pwa-common/helpers/config';
 const { colors, variables } = themeConfig;
```

Comment:
colors is no longer used and should be removed.

Reply:
Done.